src/lib/moneyHelper.ts

```diff
@@ -0,0 +1,13 @@
+interface MoneyField {
+  amount: number
+  currencyCode: String
+}
+
+export const moneyFieldToUnit = (moneyField: MoneyField) => {
+  switch (moneyField.currencyCode) {
```

Comment:
Doesn't have to be a part of this PR, but I worry that this is too restrictive and the source of the constraint is unclear.

You may just want to allow all currencies put a comment describing that we don't currently support currencies without cents conversions.

Reply:
alright, going to make this change, going to add comment saying we only support currencies with cent and once we start support currencies without cents we need to add that exception here.